### PR TITLE
Doc-Bug-Kubernetes-Install

### PR DESCRIPTION
**Pull request by bmckercher123**
User sent email regarding error in docs: Calico, Kubernetes Install, 50 nodes or more. 

Per slack discussion with ET, Lance, KP, I made the following changes. 
Replaced "calico-typha.yaml" and "calico-etcd.yaml" in etcd section with calico.yaml.

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit c25039d1b46d29aa7043c11c3f7da1e0e56b2492

https://deploy-preview-2674--calico.netlify.com

File: master/getting-started/kubernetes/installation/calico.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ datastore type and number of nodes.
 1. Download the {{site.prodname}} networking manifest for the Kubernetes API datastore.
 
    ```bash
-   curl {{site.url}}/{{page.version}}/manifests/calico-typha.yaml -O
+   curl {{site.url}}/{{page.version}}/manifests/calico.yaml -O
```
Comment (caseydavenport) [inline master/getting-started/kubernetes/installation/calico.md]: I don't think this is the right fix - this changes the manifest that is downloaded to be a different one.

We could either update this line below: https://github.com/projectcalico/calico/pull/2674/files#diff-4206179465750ddee2eb8085ba8f2fb2R85

Or change to this:

```
curl {{site.url}}/{{page.version}}/manifests/calico-typha.yaml -o calico.yaml
```

Comment (bmckercher123) [reply master/getting-started/kubernetes/installation/calico.md]: The discussion is here: https://tigera.slack.com/messages/C0FQH92CA/convo/C2PQSHPCG-1560870200.169500/?cdn_fallback=1

I'm confused then.

Comment (bmckercher123) [reply master/getting-started/kubernetes/installation/calico.md]: Updated curl command.

**Pull request merged**